src/SearchComponents/SearchUtility.js

```diff
@@ -3,6 +3,29 @@ import {json} from '../store.js';
 import {get} from 'svelte/store';
 const jVal = get(json);
 let ontology = jVal['DSP']['Ontology'];
+let shortName = jVal['DSP']['ShortName'];
+export function getPropString(prop){
+    let toReturn = '';
+    let curr = prop;
+
+    let linkStack = [];
+    while(curr){
+        linkStack.push(curr);
+        curr = curr['linkedVia'];
+    }
+    let parent = '?mainres';
+    let currProp;
+    while (linkStack.length > 0){
+        currProp = linkStack.pop();
+        if ("incoming" in currProp && currProp["incoming"] === "true"){
```

Comment:
`currProp["incoming"] === "true"` can be shortened to `currProp["incoming"]`

Reply:
As it is "true" rather than the boolean value, I'll leave it for the moment. The incoming link solution willl need a rework in the json at some point anyway